### DISCUSSION TITLE
[Bug] Cannot manager users on admin page

### DISCUSSION THREAD

**Discussion by RainCat1998** [dani-garcia/vaultwarden#4136] [Q&A]
### Deployment environment

* vaultwarden version: 1.30.1

* Install method: docker-compose

* Clients used:  web vault

* Reverse proxy and version:  nginx

* MySQL/MariaDB or PostgreSQL version: 

* Other relevant details:

### Steps to reproduce

1. Open the admin page and go to the 'Users' tab.
2. Click the buttons on the 'Action' row.

### Expected behavior

Popup a confirmation window.

### Actual behavior

Nothing happens.

### Troubleshooting data

![image](https://github.com/dani-garcia/vaultwarden/assets/36730607/d0617a2b-80c5-474f-b443-23e0085cbe93)

**Comment by BlackDex** [comment]
You probably override some headers in your reverse proxy config which causes the scripts to not work. Like CSP for example.
